src/ui/public/directives/kbn_top_nav.js

```diff
@@ -26,6 +26,8 @@ module.directive('kbnTopNav', function (Private) {
       label: _.capitalize(opt.key),
       hasFunction: !!opt.run,
       description: ('Toggle ' + opt.key),
+      noButton: !!opt.noButton,
+      openByDefault: !!opt.openByDefault ,
```

Comment:
Discussed on irc, but it is available to the scope.
https://github.com/elastic/kibana/pull/6657/files#diff-5616a348aebd0b4a63f32be784d7432cR96